src/get-started/HelloWorld.java

```diff
@@ -0,0 +1,7 @@
+
+public final class Run {
```

Comment:
why do we need this?

Reply:
Well, it is a prototype: https://github.com/exonum/exonum-doc/pull/403/commits/b02a6ba964d18d046a94a9e8efd56313fb9f044a